content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -417,22 +417,7 @@ const result = await prisma.post.findMany({
 
 ## Nested writes
 
-A nested write allows you to write **relational data** to your database in **a single transaction**. For example, the following nested write creates a `User` and two related `Post` records:
-
-```ts
-const createUserAndPost = await prisma.user.create({
-  data: {
-    email: 'elsa@prisma.io',
-    name: 'Elsa Prisma',
-    posts: {
-      create: [
-        { title: 'How to make an omelette' },
-        { title: 'How to eat an omelette' },
-      ],
-    },
-  },
-})
-```
```

Comment:
Intentionally removed this full example?

Reply:
(This unforutnately makes Percy and Lostpixel useless to review the rest of the changes)

Reply:
Yes. The example was duplicated. I've re-ordered the content in this section.
You can find the example from line 435 (updated content). _GH couldn't let me grab the line number 😔 _

Reply:
I only cleaned up the example and explanation in this section 

<img width="489" alt="CleanShot 2023-04-21 at 12 53 25@2x" src="https://user-images.githubusercontent.com/33921841/233618870-0c2a75ca-a218-47ca-9b94-9121ae1368ef.png">